hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/cert/ListSubcommand.java

```diff
@@ -89,12 +104,31 @@ protected void execute(SCMSecurityProtocol client) throws IOException {
     HddsProtos.NodeType nodeType = parseCertRole(role);
     List<String> certPemList = client.listCertificate(nodeType,
         startSerialId, count, isRevoked);
-    LOG.info("Certificate list:(Type={}, BatchSize={}, CertCount={})",
-        type.toUpperCase(), count, certPemList.size());
     if (count == certPemList.size()) {
       LOG.info("The certificate list could be longer than the batch size: {}." +
```

Comment:
Please output this to err, else json parse will fail if this prints along with json.